src/app/components/hris/employees/employee-profile/accordions/accordion-banking/accordion-banking.component.ts

```diff
@@ -164,53 +164,58 @@ export class AccordionBankingComponent {
   }
 
   saveBankingDetails() {
-    this.editBanking = false;
-    this.isUpdated = true;
-    const employeeBankingFormValue = this.employeeBankingsForm.value;
-    this.employeeBankingDto = {
-      id: this.bankingId,
-      employeeId: this.employeeProfile?.id,
-      bankName: employeeBankingFormValue.bankName,
-      branch: `${employeeBankingFormValue.branch}`,
-      accountNo: `${employeeBankingFormValue.accountNo}`,
-      accountType: employeeBankingFormValue.accountType,
-      status: 1,
-      declineReason: this.bankingReason,
-      file: employeeBankingFormValue.file
-    }
-    if (this.hasBankingData) {
-      this.employeeBankingService.updatePending(this.employeeBankingDto).subscribe({
-        next: () => {
-          this.snackBarService.showSnackbar("Updated", "snack-success");
-          this.getEmployeeBankingData();
-          this.checkBankingInformationProgress();
-          this.totalBankingProgress();
-          this.employeeBankingStarterkitService.getAllBankingAndStarterkits();
-          this.hasUpdatedBanking = true;
-          this.editBanking = false;
-          this.employeeBankingsForm.disable();
-        },
-        error: (error) => {
-          this.snackBarService.showSnackbar("Unable to Save Banking Information", "snack-error");
-        }
-      })
+    if(this.bankingPDFName.length >= 1){
+      this.editBanking = false;
+      this.isUpdated = true;
+      const employeeBankingFormValue = this.employeeBankingsForm.value;
+      this.employeeBankingDto = {
+        id: this.bankingId,
+        employeeId: this.employeeProfile?.id,
+        bankName: employeeBankingFormValue.bankName,
+        branch: `${employeeBankingFormValue.branch}`,
+        accountNo: `${employeeBankingFormValue.accountNo}`,
+        accountType: employeeBankingFormValue.accountType,
+        status: 1,
+        declineReason: this.bankingReason,
+        file: employeeBankingFormValue.file
+      }
+      if (this.hasBankingData) {
+        this.employeeBankingService.updatePending(this.employeeBankingDto).subscribe({
+          next: () => {
```

Comment:
Can the code in the `next()` of adding and updating banking information not be placed in a function to call? Seems to me similar set of code.